odoo_initializer/models/drug_loader.py

```diff
@@ -4,15 +4,8 @@
 class DrugLoader(BaseLoader):
     model_name = "product.template"
     field_mapping = {
-        "lst_price": "odoo_price",
-        "product_variant_ids/categ_id/id": "odoo_category",
-        "type":"odoo_type",
-        "name": "Name",
-        "product_variant_ids/uuid": "Uuid",
+        "product_variant_ids/id": "odoo_id",
```

Comment:
Should we make the change and support 2 non-identical fields?
Maybe `odoo_variant_id`